source/code-snippets/example-if.liquid

```diff
@@ -1,29 +1,31 @@
 {% codeExample 'example-if' %}
-@mixin avatar($size, $circle: false) {
-  width: $size;
-  height: $size;
+  @use "sass:math";
 
-  @if $circle {
-    border-radius: $size / 2;
+  @mixin avatar($size, $circle: false) {
+    width: $size;
+    height: $size;
+
+    @if $circle {
+      border-radius: math.div($size, 2);
+    }
   }
-}
 
-.square-av {
-  @include avatar(100px, $circle: false);
-}
-.circle-av {
-  @include avatar(100px, $circle: true);
-}
-===
-@mixin avatar($size, $circle: false)
-  width: $size
-  height: $size
+  .square-av {
+    @include avatar(100px, $circle: false);
+  }
+  .circle-av {
+    @include avatar(100px, $circle: true);
+  }
+  ===
+  @mixin avatar($size, $circle: false)
```

Comment:
I think we'll also want `@use "sass:math"` in the Sass section

Reply:
fixed in https://github.com/oddbird/sass-site/commit/a331c7299a0bac6107d286de67e37182b3d80abf